examples/asr/transcribe_speech.py

```diff
@@ -298,8 +309,14 @@ def main(cfg: TranscriptionConfig) -> Union[TranscriptionConfig, List[Hypothesis
         else:
             cfg.decoding = cfg.rnnt_decoding
 
-    # prepare audio filepaths and decide wether it's partial audio
-    filepaths, partial_audio = prepare_audio_data(cfg)
+    if cfg.use_model_transcribe:
```

Comment:
This should use new flag (use model transcribe should be enabled by default or rename flag and make it so that  model.transcribe() is the default. 

Actually i. want to avoid this flag entirely, its just so confusing. This is a transcribe speech wrapper script, why should we even have to specify whether we want to call the models transcribe or not ?

Reply:
CC @stevehuang52

Reply:
removed and added a special case for multi-task AED models...